docs/source/configuration/open_edx.rst

```diff
@@ -176,6 +177,9 @@ In Open edX (derived from instructions `here <https://edx.readthedocs.io/project
 
     * Linux users: The gateway IP of the docker-compose networking setup for mitxonline as found via ``docker network inspect mitx-online_default``
     * OSX users: Use ``host.docker.internal``
+    
+  * Save the configuration.  Then, while on the ``/admin/third_party_auth/oauth2providerconfig/`` page, select 'update' for the configuration you just created.
+    * Check the 'enabled' box at the top, and save the configuration.
```

Comment:
I think this could be a redundant step, We have the `Enabled` checkbox while creating the configuration for the first time and the configuration doesn't really have to be created and then updated to check that box.

 Instead, I think we can add a bullet [here](https://github.com/mitodl/mitxonline/pull/711/files#diff-a90f6fb11bd555c7c3cf903cafc0798bf2abc9e796c6dfce046f9ffba5fbbd0dR160) for the `Enabled` checkbox where we mention about other checkboxes.